Makefile

```diff
@@ -120,7 +120,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build:
-	docker build -t ${IMG} .
+	docker build -t ${IMG} --build-arg OPERATOR_VERSION=${VERSION} .
```

Comment:
This will only be applied when building the docker image but not when debugging locally  via `make run` - maybe we can fix that by setting the variable when doing `go run` as well?